### PR TITLE
user input for PRIME_LIMIT server.py

### PR DESCRIPTION
**Pull request by REGATTE**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -8,25 +8,25 @@
 
 # Import python socket, random, sympy, hashlib, pickle module
 import socket, random, sympy, hashlib, pickle
-
+print("initialising model")
```
Comment (AbhinavRobinson) [inline server.py:11]: remove this

Comment (AbhinavRobinson) [inline server.py]: remove this

File: server.py [hunk 2]
```diff
@@ -8,25 +8,25 @@
...
 
 # Connect and recieve message
 def recieve():
 
     # Define prime upper bound
-    PRIME_LIMIT = 1000000000000
-
+    #PRIME_LIMIT = 1000000000000
+    PRIME_LIMIT = int(input("enter a int value to mark the upper bound"))
```
Comment (AbhinavRobinson) [inline server.py]: okay

File: server.py [hunk 3]
```diff
@@ -8,25 +8,25 @@
...
     # Generate Key Pair
     e,d,n = generate(sympy.randprime(1,PRIME_LIMIT),sympy.randprime(1,PRIME_LIMIT))
 
     print(f"Encryption Key: {e} \n Mod Key: {n} \n Secret Key: {d}")
 
     # Initialize socket stream
     s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-
+    print("socket is initialising")
```
Comment (AbhinavRobinson) [inline server.py:28]: okay

**Review by AbhinavRobinson — CHANGES_REQUESTED** [review]

**Review by aniketbiprojit — COMMENTED** [review]
Nahi manta kya.

**Comment by AbhinavRobinson** [comment]
Done.

File: server.py [hunk 4]
```diff
@@ -9,24 +9,27 @@
...
 
-
 # Connect and recieve message
 def recieve():
 
-    # Define prime upper bound
-    PRIME_LIMIT = 1000000000000
-
+    # Take prime upper bound
+    PRIME_LIMIT = int(input("Enter PRIME the upper bound (1000,1000000000): "))
```
Comment (AbhinavRobinson) [inline server.py:16]: Should be removed later, not necessary.

**Review by AbhinavRobinson — COMMENTED** [review]
Retrospect

**Pull request merged**